openpype/hosts/maya/api/plugin.py

```diff
@@ -143,16 +143,49 @@ def load(
         assert os.path.exists(self.fname), "%s does not exist." % self.fname
 
         asset = context['asset']
+        subset = context['subset']
+        settings = get_project_settings(context['project']['name'])
+        custom_naming = settings['maya']['load']['reference_loader']
         loaded_containers = []
 
-        count = options.get("count") or 1
-        for c in range(0, count):
-            namespace = namespace or lib.unique_namespace(
-                "{}_{}_".format(asset["name"], context["subset"]["name"]),
-                prefix="_" if asset["name"][0].isdigit() else "",
-                suffix="_",
+        if not custom_naming['namespace']:
+            raise ValueError("No namespace specified")
+        elif not custom_naming['group_name']:
+            raise ValueError("No group name specified")
```

Comment:
Should this error message be more descriptive errors, like at least mention that it's based on project settings or alike? Or is this fine as is?

Also should this be `openpype.pipeline.load.LoadError`?

Reply:
Yeah, lets get it more descriptive.

@iLLiCiTiT do you have an input on what error to use?

Reply:
The difference is how it's displayed to user. `LoadError` should show only the error message without traceback, so `LoadError` is suggested.

Reply:
```suggestion
        if not custom_naming['namespace']:
            raise LoadError("No namespace specified")
        elif not custom_naming['group_name']:
            raise LoadError("No group name specified")
```